packages/components/src/experience-control/icons.tsx

```diff
@@ -1,47 +1,32 @@
-export const IconGood = () => {
-	return (
-		<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-			<g id="Icon">
-				<path
-					id="Shape"
-					fill-rule="evenodd"
-					clip-rule="evenodd"
-					d="M5.99452 13.0808C6.89004 13.0808 7.616 12.3549 7.616 11.4594C7.616 10.5638 6.89004 9.83789 5.99452 9.83789C5.09901 9.83789 4.37305 10.5638 4.37305 11.4594C4.37305 12.3549 5.09901 13.0808 5.99452 13.0808ZM18.0054 13.0808C18.901 13.0808 19.6269 12.3549 19.6269 11.4594C19.6269 10.5638 18.901 9.83789 18.0054 9.83789C17.1099 9.83789 16.384 10.5638 16.384 11.4594C16.384 12.3549 17.1099 13.0808 18.0054 13.0808ZM15.352 14.5221C13.6377 16.7223 10.493 17.1864 8.84968 14.7405C8.49299 14.2091 8.01618 14.0562 7.41927 14.2819C7.31736 14.3183 7.23185 14.382 7.16269 14.473C6.71865 15.0517 6.73138 15.6213 7.2009 16.1818C9.90335 19.4193 14.604 19.3756 17.0007 15.8542C17.4411 15.21 17.3137 14.7132 16.6186 14.3638L16.4439 14.2764C16.0107 14.0581 15.6468 14.1399 15.352 14.5221Z"
-					fill="#255C33"
-				/>
-			</g>
-		</svg>
-	);
-};
+export const IconGood = () => (
+	<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M5.99452 13.0808C6.89004 13.0808 7.616 12.3549 7.616 11.4594C7.616 10.5638 6.89004 9.83789 5.99452 9.83789C5.09901 9.83789 4.37305 10.5638 4.37305 11.4594C4.37305 12.3549 5.09901 13.0808 5.99452 13.0808ZM18.0054 13.0808C18.901 13.0808 19.6269 12.3549 19.6269 11.4594C19.6269 10.5638 18.901 9.83789 18.0054 9.83789C17.1099 9.83789 16.384 10.5638 16.384 11.4594C16.384 12.3549 17.1099 13.0808 18.0054 13.0808ZM15.352 14.5221C13.6377 16.7223 10.493 17.1864 8.84968 14.7405C8.49299 14.2091 8.01618 14.0562 7.41927 14.2819C7.31736 14.3183 7.23185 14.382 7.16269 14.473C6.71865 15.0517 6.73138 15.6213 7.2009 16.1818C9.90335 19.4193 14.604 19.3756 17.0007 15.8542C17.4411 15.21 17.3137 14.7132 16.6186 14.3638L16.4439 14.2764C16.0107 14.0581 15.6468 14.1399 15.352 14.5221Z"
+			fill="#255C33"
```

Comment:
Since this scatters the color variable management, could we maybe use `currentColor` for the fills here, and control the actual color with the `color` property in the stylesheet?